dev-tex/biber/biber-2.7.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit perl-module
+
+DESCRIPTION="A BibTeX replacement for users of biblatex"
+HOMEPAGE="http://biblatex-biber.sourceforge.net/ https://github.com/plk/biber/"
+SRC_URI="https://github.com/plk/biber/archive/v${PV}.tar.gz  -> ${P}.tar.gz"
+
+LICENSE="|| ( Artistic-2 GPL-1 GPL-2 GPL-3 )"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="doc test"
+
+RDEPEND=">=dev-lang/perl-5.24
+	dev-perl/autovivification
+	dev-perl/Business-ISBN
+	dev-perl/Business-ISMN
+	dev-perl/Business-ISSN
+	dev-perl/Class-Accessor
+	dev-perl/Data-Compare
+	dev-perl/Data-Dump
+	dev-perl/Data-Uniqid
+	dev-perl/DateTime-Calendar-Julian
+	dev-perl/DateTime-Format-Builder
+	dev-perl/Encode-EUCJPASCII
+	dev-perl/Encode-HanExtra
+	dev-perl/Encode-JIS2K
+	dev-perl/File-Slurp
+	dev-perl/IPC-Run3
+	dev-perl/libwww-perl[ssl]
+	>=dev-perl/Lingua-Translit-0.250
+	dev-perl/List-AllUtils
+	dev-perl/List-MoreUtils
```

Comment:
Thanks for spotting this. List-MoreUtils-XS was installed through some other dependency on my machine, so I've added it to the list.